### ISSUE TITLE
Remove Image button fails without error message

### ISSUE THREAD

**Issue by styfle** [kevana/ui-for-docker#195]
First of all, great work with this dockerui app! :tada: 

I found a bug when clicking on the "Remove Image" button.

## web console
```
Error: [$resource:badcfg] Error in resource configuration for action `remove`. Expected response to contain an array but got an object
```

## Request
```
Request URL:http://localhost:9000/dockerapi/images/sha256:e713b5e836513871b647b658cd09a78537b47375f5f1402f7085ac773ec629c4
Request Method:DELETE
Status Code:200 OK
```

## Response
```
conflict: unable to delete e713b5e83651 (must be forced) - image is being used by stopped container e96cb9bcc278
```

I believe the bug is cause by the string being returned when the client is [expecting an array](https://github.com/crosbymichael/dockerui/blob/6963a1ae8adaeae6c7018176a06550c5cc239f19/app/shared/services.js#L97).

However I'm not familiar enough with the code to fix it at this time.

**Comment by kevana** [comment]
What DockerUI version are you running? If it's not the latest, give  `1.10.0-beta` a try, f353dc2c419664bc8aa5523bad6d9af7025106ee should've resolved this.

**Comment by styfle** [comment]
I was running `latest` tag.
I don't see a `1.10.0-beta` but I tried with `0.10.0-beta` and got the same problem using Action > Remove since there is no longer a Remove Image button.

**Comment by kevana** [comment]
Whoops, I meant `0.10.0-beta`. There should still be a Remove Image button on the bottom of image pages:
<img width="890" alt="screen shot 2016-03-25 at 10 12 25 am" src="https://cloud.githubusercontent.com/assets/981551/14046873/225ea3da-f272-11e5-82b3-fb72b778c72c.png">

But I'm seeing the same error, the global interceptor didn't catch this because it checks for `Conflict.` case sensitively :disappointed:

**Comment by chiu0602** [comment]
In the fix [f353dc2](https://github.com/kevana/ui-for-docker/commit/f353dc2c419664bc8aa5523bad6d9af7025106ee), `response.data.startsWith('Conflict.')` returns false for response `conflict: unable to delete e713b5e83651 (must be forced) - image is being used by stopped container e96cb9bcc278`. I met with the same issue on Docker API Version: 1.23 UI Version: v0.10.1-beta.

By the way, thanks for great dockerui app!

**Comment by kevana** [comment]
Fixed by #211
